ProteoformSuiteGUI/ExperimentTheoreticalComparison.cs

```diff
@@ -385,6 +385,7 @@ private void nUD_PeakCountMinThreshold_ValueChanged(object sender, EventArgs e)
                 Parallel.ForEach(SaveState.lollipop.et_peaks, p =>
                 {
                     p.Accepted = p.peak_relation_group_count >= SaveState.lollipop.min_peak_count_et;
+                    p.count_nearby_decoys(SaveState.lollipop.ed_relations.Values.SelectMany(v => v).ToList());
```

Comment:
I think the issue was I had to find which relations were within the range of each peak.... maybe the first/only time I count, I'll set the decoy relation's peak to be that peak (that the relation is in the range of). Then if the peak acceptance changes, I'll parallel loop through and change the acceptance of relations based on the acceptance of peaks